src/main/msp/msp.c

```diff
@@ -1130,6 +1130,7 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
     case MSP_GPS_RESCUE:
         sbufWriteU16(dst, gpsRescueConfig()->angle);
         sbufWriteU16(dst, gpsRescueConfig()->initialAltitudeM);
+        sbufWriteU16(dst, gpsRescueConfig()->targetLandingAltitudeCm);
```

Comment:
Changes to the MSP structure are not allowed. New parameters must be appended to the end of the message.

Reply:
Done